.github/workflows/run_semgrep_scan.yml

```diff
@@ -0,0 +1,17 @@
+# doc https://semgrep.dev/docs/semgrep-ci/sample-ci-configs/#sample-github-actions-configuration-file
+name: Run Semgrep Scan
+
+on:
+  workflow_dispatch:
+    inputs:
+      verbose:
+        description: verbose flag for semgrep scan
+        type: boolean
+        default: false
+
+jobs:
+  semgrep:
+    uses: ./.github/workflows/semgrep.yml
+    with:
+      verbose: ${{ inputs.verbose }}
+    secrets: inherit
```

Comment:
Should pass only one secret `SEMGREP_APP_TOKEN` not inherit all.

Reply:
we use `secrets: inherit` in many locations such as prod_release. What's bad about this inherit?

Reply:
It is about security issues by sending all secret keys or tokens to third-party services.

Reply:
I read the document GitHub limit accessing secret in the organization or enterprise scope, so we can use inherit.